src/main/java/commands/AddAuthorCommand.java

```diff
@@ -17,7 +17,7 @@ public AddAuthorCommand(String userInput) {
 
     @Override
     public void execute(Ui ui, AuthorList authorList) throws TantouException {
-        String authorName = parser.getAuthorName(userInput);
+        String authorName = parser.getAuthorNameFromInput(userInput);
```

Comment:
Love the change! Greatly improves clarity on what the method is supposed to do just from the name change :)